composer.json

```diff
@@ -15,10 +15,10 @@
     ],
     "require": {
         "php": "^7.1",
-        "illuminate/support": "5.5.* || 5.6.* || 5.7.* || 5.8.*"
+        "illuminate/support": "^5.7 || ^6.0"
     },
     "require-dev": {
-        "orchestra/testbench": "~3.0",
+        "orchestra/testbench": "~3.0 || ^4.0",
```

Comment:
Same here.

As `4.0` is only required from Laravel 6.0, it makes more sense to only require it and tag it as a major release.